lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -491,7 +491,7 @@ protected static function get_style_nodes( $theme_json, $selectors = array() ) {
 
 		if ( isset( $theme_json['styles']['elements'] ) ) {
 			foreach ( self::ELEMENTS as $element => $selector ) {
-				if ( ! isset( $theme_json['styles']['elements'][ $element ] ) ) {
+				if ( ! isset( $theme_json['styles']['elements'][ $element ] ) || empty( static::ELEMENTS[ $element ] ) ) {
```

Comment:
This will produce an `Undefined index` in PHP 5.6 - https://3v4l.org/vqKpS#v5.6.40, and we cannot use `isset()` since it will result in a fatal error.

So let's use `array_key_exists`, similar to #42567.

Reply:
Thanks @Mamaduka 

Just checking, do you mean the `isset( $theme_json['styles']['elements'][ $element ] )` or `empty( static::ELEMENTS[ $element ] )`? 

Or isn't `empty` okay with constants either?

I'll update to 

`if ( ! isset( $theme_json['styles']['elements'][ $element ] ) || array_key_exists( $element, static::ELEMENTS ) ) {`

Reply:
Sorry for not being clear; it's still early morning for me ☕

I mean - `empty( static::ELEMENTS[ $element ] )`

The `isset` and `empty` don't work as you expect with class constants in PHP 5.6.

Reply:
Ah great. I knew about isset, but TIL about empty. Thanks a lot for the nudge. 

Updated.

Reply:
Thanks for the quick follow-up.